website/docs/components/form/layout/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,125 @@
+## Usage
+
+Form layout components in Ember and their related Layout “Template” component in Figma are structural components that provide the scaffolding for building consistent forms across products. These components establish spacing and layout standards that give users an experience focused on simplicity and ease of use.
+
+### When to use
+
+- To structure and organize form elements and inputs into a consistent layout
+- When creating a form of any size in our products, including forms nested within Flyout or Modal components
+
+### When not to use
+
+- To organize non-form content
+- To create key-value pairs, use the [Key Value Inputs](/components/forms/key-value-inputs) component
+
+## Single column layout
+
+The Form component ensures a consistent and streamlined user experience. We recommend using a single-column layout for forms, as multi-column layouts can disrupt the visual flow and reading order. Multi-column layouts can potentially result in missed inputs requiring resolution and thereby increasing friction. They should only be used when grouping related inputs improves the user experience.
+
+![The anatomy of the Form layout, indicating the header, sections, separator, and actions](/assets/components/form/layout/form-anatomy.png)
+
+### Spacing
+
+Spacing in forms is based on a decreasing scale on an 8px grid. From largest to smallest, the spacing structure is:
+
+- Sections are 32px apart
+- Separators are 32px from Section elements on either vertical side
+- Inputs within a Section are 24px apart; unless they are within a Section Field Group
+- Inputs within a Section Field Group have 16px spacing in wider viewports. In small viewports, the stacking matches the 24px spacing that all other Section elements share
+- Text elements inside the Form Header and Form Section Header are 8px apart
+
+![Representation of the various spacing sizes between Form layout components](/assets/components/form/layout/form-layout-spacing.png)
+
+!!! Do
+
+Organize inputs together when they are closely related and beneficial to read together. No more than three inputs should be put within a single row.
+
+![Three credit card information fields arranged with one full width field followed by a row of two side-by-side equal width fields](/assets/components/form/layout/form-do-rows.png)
+
+!!!
+
+!!! Dont
+
+Place inputs in rows to save space or create columns within the form.
+
+![Three equal width fields for credit card information arranged in a single row](/assets/components/form/layout/form-dont-rows.png)
+
+!!!
+
+Visit the [Form Patterns][/patterns/form-patterns] guidelines for additional details on spacing in forms.
```

Comment:
I can help add a new brief section for the Form component here above the Form Header section. (I recommend making the other recommended changes above first since they will help restructure other content coming before this.)

Reply:
Added a suggested edit to apply.